### PR TITLE
155889952 Create java classes

### PR DESCRIPTION
**Pull request by simiyu1**
#### What does this PR do?
 - Create java classes for data, response, and users
 - Structure the files in MVP design

#### Description of Task to be completed?
 - Create directories service, model, view, and presenter
 - Create classes GitUser, Data, and RestResponse

#### How should this be manually tested?
 - Navigate into the Java directory from the root, inside it you will find 
    service, model, view, and presenter directories
 - Navigate to the model directory, inside it you will find the classes GitUser, Data, and RestResponse

#### What are the relevant pivotal tracker stories?
 - [#155889952](https://www.pivotaltracker.com/story/show/155889952)

### CODE REVIEW

**Comment by Rwothoromo** [comment]
@simiyu1 Could we have one commit?

File: app/src/main/java/com/example/simiyunabende/javadevs/model/GithubUsers.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package com.example.simiyunabende.javadevs.model;
+
+import com.google.gson.annotations.SerializedName;
+
+//public class GitUser {
```
Comment (Rwothoromo) [inline app/src/main/java/com/example/simiyunabende/javadevs/model/GithubUsers.java]: @simiyu1 This comment is unnecessary, could you remove it?

Comment (simiyu1) [reply app/src/main/java/com/example/simiyunabende/javadevs/model/GithubUsers.java]: Thank you, resolved.

**Comment by Rwothoromo** [comment]
@simiyu1 Do merge.

**Pull request merged**